src/sqlfluff/cli/commands.py

```diff
@@ -695,154 +704,67 @@ def do_fixes(lnt, result, formatter=None, **kwargs):
     return False  # pragma: no cover
 
 
-@cli.command(cls=DeprecatedOptionsCommand)
```

Comment:
@alanmcruickshank: Can you suggest how to review this? It looks like a lot of changes starting here, but I'm guessing some of this was code reorg?

Reply:
Good point. I'll add a summary in the PR above.